apps/financial-aid/web-veita/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,88 @@
+import NextAuth, { Callbacks } from 'next-auth'
```

Comment:
Should this be in a lib (apps/financial-aid/web-osk/pages/api/auth/[...nextauth].ts)

Reply:
Could you perhaps explain better? What do you want in a lib? The whole api route/s?

Reply:
The content of this file handles all auth related (and is most likely valuable to other projects). Its content is the exact same as `apps/financial-aid/web-osk/pages/api/auth/[...nextauth].ts`.
Wondering if this file should just call something from a library, thus both files can share the same library and any additions/fixes will only be made in a single place.

Reply:
Gotcha, I extracted some of the functionality into `libs/financial-aid/shared/src/lib/identityProvider.ts`. But you want to go even further and create something like `libs/next-auth` ? 

I am all for it, just verifying before diving in.

Reply:
Yeah, if we want to do that.. let's do it in a follow-up PR 👍